chart/templates/heartbeat-rbac.yaml

```diff
@@ -0,0 +1,48 @@
+{{with .Values -}}
+---
+###
+### Heartbeat RBAC
+###
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: linkerd-heartbeat
+  namespace: {{.Namespace}}
+  labels:
+    {{.ControllerNamespaceLabel}}: {{.Namespace}}
+rules:
+- apiGroups: [""]
+  resources: ["configmaps"]
+  verbs: ["get"]
+  resourceNames: ["linkerd-config"]
+- apiGroups: [""]
+  resources: ["namespaces"]
```

Comment:
Could we omit this privilege if we used the creation date of the configmap to determine the age of the install instead of the creation date of the namespace?

Reply:
Great idea! Updated.